hadoop-hdfs-project/hadoop-hdfs-client/src/main/java/org/apache/hadoop/hdfs/DFSUtilClient.java

```diff
@@ -733,13 +733,13 @@ public static boolean isLocalAddress(InetSocketAddress targetAddr)
     InetAddress addr = targetAddr.getAddress();
     Boolean cached = localAddrMap.get(addr.getHostAddress());
     if (cached != null) {
-      LOG.trace("Address {} is {} local", targetAddr, (cached ? "" : "not"));
+      LOG.trace("Address " + targetAddr + (cached ? " is local" : " is not local"));
```

Comment:
Why are you reverting back from logger format to normal string concat. better not to ditch the {}

Reply:
you are right, @ayushtkn , but the corrected format seems not that perfect. do you have better ideas?

Reply:
The present change Looks good to me, What is the problem in that?

Reply:
> 

just a feeling ,not that perfect ,but current change is a relative simple fix way. btw, we don't need to add the test case for this kind of log info fix ,right ? @ayushtkn

Reply:
Yeps, not required